HelpSource/Reference/SCDocStyling.schelp

```diff
@@ -0,0 +1,46 @@
+title:: SCDoc Styling
+summary:: Help system CSS styles customization and themes
+categories:: HelpSystem
+related:: Guides/WritingHelp, Classes/SCDoc, Classes/SCDocHTMLRenderer
+
+Documentation rendered HTML reads the global style from teletype::scdoc.css::,
+but also reads teletype::frontend.css:: and teletype::custom.css:: (in that
+order) if available, to enable specific frontends and users to override the CSS.
```

Comment:
This sentence is a bit complicated. New sentence for 'To enabled ...' maybe? Then you can remove the brackets.

Reply:
So something like this?

> Documentation rendered HTML reads the global style from teletype::scdoc.css::,
but also reads teletype::frontend.css:: and teletype::custom.css:: (in that
order) if available.
> This is to enable specific frontends and allow users to override the CSS.